src/main/java/school/hei/haapi/endpoint/event/EventProducer.java

```diff
@@ -32,7 +32,7 @@
   public static class Conf {
 
     private final Region region;
-    private static final int MAX_PUT_EVENT_ENTRIES = 10;
+    public static final int MAX_PUT_EVENT_ENTRIES = 10;
```

Comment:
Bad. Anytime we need to make this info public is bad... I just saw that we actually don't do anything for ensuring that PUT /users does not do the check. Let's also do nothing for adding fees (for the moment) **BUT just make feeService::saveAll to be @Transactional**

Reply:
Actually, PUT /users checks the payload before sending the request to Cognito. So, when the error occurs, Postgresql rolls back the data